sktime/transformations/series/window_summarizer.py

```diff
@@ -143,17 +143,9 @@ class LaggedWindowSummarizer(BaseTransformer):
     """
 
     _tags = {
-        "scitype:transform-input": "Series",
-        "scitype:transform-output": "Series",
-        # "scitype:transform-labels": "None",
-        "scitype:instancewise": True,  # is this an instance-wise transform?
-        # "X_inner_mtype": ["pd.DataFrame"
-        # # which mtypes do _fit/_predict support for X?
-        # # X_inner_mtype can be Panel mtype even if
-        # transform-input is Series, vectorized
-        # "y_inner_mtype": "None",  # which mtypes do _fit/_predict support for y?
-        "capability:inverse_transform": False,  # does transformer
-        # have inverse transform
+        "scitype:transform-output": "Panel",
```

Comment:
why that? I thought `Series` was right, can you explain?

Reply:
yes that is correct, but then tests fail currently due to this self._convert_X issue. But probably not the right way to fix I agree. will change it back, just wanted to see if everything else works.

Reply:
I was just working on it, give me a sec

Reply:
FYI, here: https://github.com/alan-turing-institute/sktime/pull/1977

Thanks for creating the very specific condition that caused the failure 😄

Reply:
works now